x-pack/plugins/apm/public/components/app/TransactionOverview/__jest__/TransactionOverview.test.js

```diff
@@ -4,6 +4,27 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+jest.mock(
+  'ui/chrome',
+  () => ({
+    getUiSettingsClient: () => {
+      return {
+        get: key => {
+          switch (key) {
+            case 'timepicker:timeDefaults':
+              return { from: 'now-15m', to: 'now', mode: 'quick' };
+            case 'timepicker:refreshIntervalDefaults':
+              return { display: 'Off', pause: false, value: 0 };
+            default:
+              throw new Error(`Unexpected config key: ${key}`);
+          }
+        }
+      };
+    }
+  }),
+  { virtual: true }
+);
+
```

Comment:
Instead of adding the mocks for `ui/chrome` separately, we could add them to either `kibana/x-pack/plugins/__mocks__/ui/chrome.js` (shared for all x-pack plugins) or `kibana/x-pack/plugins/apm/__mocks__/ui/chrome.js` (just apm plugin) like:

```js
function getUiSettingsClient() {
  return {
    get: key => {
      switch (key) {
        case 'timepicker:timeDefaults':
          return { from: 'now-15m', to: 'now', mode: 'quick' };
        case 'timepicker:refreshIntervalDefaults':
          return { display: 'Off', pause: false, value: 0 };
        default:
          throw new Error(`Unexpected config key: ${key}`);
      }
    }
  };
}

function addBasePath(path) {
  return '/mockBasePath/' + path;
}

export default {
  getUiSettingsClient,
  addBasePath
};
```

If you don't want to do that now, that's totally fine, and I'll get around to it at some point.

Reply:
There is already a mock ui/chrome.js test file under license_management and when I add a new one under `kibana/x-pack/plugins/__mocks__/ui/chrome.js`, the following is displayed when running

```
jest-haste-map: duplicate manual mock found:
  Module name: ui/chrome
  Duplicate Mock path: /Users/nathanreese/projects/kibana/x-pack/plugins/license_management/__mocks__/ui/chrome.js
This warning is caused by two manual mock files with the same file name.
Jest will use the mock file found in: 
/Users/nathanreese/projects/kibana/x-pack/plugins/license_management/__mocks__/ui/chrome.js
 Please delete one of the following two files: 
 /Users/nathanreese/projects/kibana/x-pack/plugins/__mocks__/ui/chrome.js
/Users/nathanreese/projects/kibana/x-pack/plugins/license_management/__mocks__/ui/chrome.js
``` 

Looks like this is going to be more involved than just creating the file `kibana/x-pack/plugins/__mocks__/ui/chrome.js`. I am going to leave as is for now.

Reply:
Okay, I'll just move it `kibana/x-pack/plugins/apm/__mocks__/ui/chrome.js` later.